src/ResourceManager/Network/Commands.Network.Test/ScenarioTests/PublicIpAddressTests.ps1

```diff
@@ -278,6 +278,61 @@ function Test-PublicIpAddressCRUD-ReverseFqdn
     }
 }
 
+<#
+.SYNOPSIS
+Tests edit the domain name label of a publicIpAddress with Iptags
+#>
+function Test-PublicIpAddressCRUD-IpTag
+{
+    # Setup
+    $rgname = Get-ResourceGroupName
+    $rname = Get-ResourceName
+    $domainNameLabel = Get-ResourceName
+    $rglocation = Get-Location ResourceManagement
```

Comment:
This isn't how to call Get-Location: it should be ```Get-Location -providerNamespace "Microsoft.Network" -resourceType "publicIpAddresses" -preferredLocation "West US"

Reply:
As discussed on IM, I do not want to go with this change because of lack of usage and adaptability in the networking.